### PR TITLE
NOJIRA | Install cnbuilder, add .nvmrc to use node 18

### PR DESCRIPTION
**Pull request by yvonnetangsu**

### CODE REVIEW

**Comment by iamrentman** [comment]
Thanks so much! I like it. @hmalcolmSquiz OK to merge this in to `develop`?

**Review by iamrentman — COMMENTED** [review]
Looks good. 🚀

**Review by iamrentman — APPROVED** [review]

**Comment by yvonnetangsu** [comment]
Thank you @iamrentman ! 🚀

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+18
```
Comment (sherakama) [inline .nvmrc:1]: https://endoflife.date/nodejs

Node 18 gets you 14 months of support
Node 20 gets you 26

Just sayn

Comment (iamrentman) [reply .nvmrc:1]: Ah, good lookin' out Shea. I'll bring this up with the Squiz folks.

Comment (yvonnetangsu) [reply .nvmrc:1]: Oh yeah I think we need to update to 20 at some point, but currently that's what the squiz devs have been using and it's in the requirements doc, so I didn't want to stir the pot too much 😂 @iamrentman @broleomo perhaps we should bring this up to Squiz at some point?

**Pull request merged**